index.js

```diff
@@ -31,7 +31,54 @@ if (!broadcaster.data) {
 }
 const broadcasterId = broadcaster.data[0].id;
 const broadcasterDisplayName = broadcaster.data[0].display_name;
+const pollingInterval = process.env.POLLING_INTERVAL ?? "1d";
 let date = new Date();
+switch (pollingInterval.substring(pollingInterval.length - 1)) {
+  case "d": // days
+    date.setDate(
+      date.getDate() -
+        parseInt(pollingInterval.substring(0, pollingInterval.length - 1)),
```

Comment:
Instead of duplicating the parseInt() call 6 times you could instead put it in a variable.